<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BookTrack</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(22, 108, 204, 255), stop:1 rgba(83, 190, 201, 255));
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="info_books_frame">
      <property name="styleSheet">
       <string notr="true">border: 2px solid rgba(255, 255, 255, 40);
background-color: rgb(170, 170, 255, 80);
border-radius: 13px;</string>
      </property>
      <layout class="QHBoxLayout" name="Layout_books_info">
       <item>
        <widget class="QLabel" name="icon_label_1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
background-color:none;
padding-left: 27 px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="res-rc.qrc">:/icons/icons_black/library_books_FILL0_wght400_GRAD0_opsz48.svg</pixmap>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text_label_1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>20</pointsize>
           <italic>false</italic>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
/*font-weight: bold;*/
font-size: 20pt;
background-color:none;
</string>
         </property>
         <property name="text">
          <string>Всего книг</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="amount_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
/*font-weight: bold;*/
font-size: 20pt;
background-color:none;
padding-left: 10px;</string>
         </property>
         <property name="text">
          <string>30</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="gap_label">
         <property name="styleSheet">
          <string notr="true">border: none;
/*font-weight: bold;*/
font-size: 20pt;
background-color:none;
padding-left: 20px;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="icon_label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
background-color:none;
padding-left: 15 px;
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="res-rc.qrc">:/icons/icons_black/attach_money_FILL0_wght400_GRAD0_opsz48.svg</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="text_label_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
/*font-weight: bold;*/
font-size: 20pt;
background-color:none;
</string>
         </property>
         <property name="text">
          <string>Общая цена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="total_price_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: none;
/*font-weight: bold;*/
font-size: 20pt;
background-color:none;
padding-left: 10px;</string>
         </property>
         <property name="text">
          <string>10 000</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="buttons_frame">
      <property name="styleSheet">
       <string notr="true">border:none;
background-color: rgb(255, 255, 255, 0);
</string>
      </property>
      <layout class="QHBoxLayout" name="Layout_buttons">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>12</number>
       </property>
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QPushButton" name="add_pushButton">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>65</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
font-size: 14pt;
background-color: rgba(255, 255, 255, 10);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;
}

QPushButton:hover{
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed{
background-color: rgba(255, 255, 255, 80);
}
</string>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
         <property name="icon">
          <iconset resource="res-rc.qrc">
           <normaloff>:/icons/icons_black/add_circle_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icons_black/add_circle_FILL0_wght400_GRAD0_opsz48.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="edit_pushButton">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>65</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
font-size: 14pt;
background-color: rgba(255, 255, 255, 10);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;
}

QPushButton:hover{
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed{
background-color: rgba(255, 255, 255, 80);
}</string>
         </property>
         <property name="text">
          <string>Редактировать</string>
         </property>
         <property name="icon">
          <iconset resource="res-rc.qrc">
           <normaloff>:/icons/icons_black/edit_note_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icons_black/edit_note_FILL0_wght400_GRAD0_opsz48.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="del_pushButton">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>65</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
font-size: 14pt;
background-color: rgba(255, 255, 255, 10);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;
}

QPushButton:hover{
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed{
background-color: rgba(255, 255, 255, 80);
}</string>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset resource="res-rc.qrc">
           <normaloff>:/icons/icons_black/delete_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icons_black/delete_FILL0_wght400_GRAD0_opsz48.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="filter_pushButton">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>65</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
font-size: 14pt;
background-color: rgba(255, 255, 255, 10);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;
}

QPushButton:hover{
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed{
background-color: rgba(255, 255, 255, 80);
}</string>
         </property>
         <property name="text">
          <string>Фильтр</string>
         </property>
         <property name="icon">
          <iconset resource="res-rc.qrc">
           <normaloff>:/icons/icons_black/filter_alt_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icons_black/filter_alt_FILL0_wght400_GRAD0_opsz48.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="analytics_pushButton">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>65</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei UI Light</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
font-size: 14pt;
background-color: rgba(255, 255, 255, 10);
border: 1px solid rgba(255, 255, 255, 40);
border-radius: 6px;
}

QPushButton:hover{
background-color: rgba(255, 255, 255, 40);
}

QPushButton:pressed{
background-color: rgba(255, 255, 255, 80);
}</string>
         </property>
         <property name="text">
          <string>Анилитика</string>
         </property>
         <property name="icon">
          <iconset resource="res-rc.qrc">
           <normaloff>:/icons/icons_black/analytics_FILL0_wght400_GRAD0_opsz48.svg</normaloff>:/icons/icons_black/analytics_FILL0_wght400_GRAD0_opsz48.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="maximumSize">
       <size>
        <width>10000</width>
        <height>10000</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft JhengHei UI Light</family>
        <pointsize>7</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableView{
border: 1px solid rgba(255, 255, 255, 50);
background-color: rgba(255, 255, 255, 50);
border-bottom-left-radius: 8px;
border-bottom-right-radius: 8px; 
}

QTableView::section{
border: none;
background-color: rgba(36, 42, 132);
color: rgba(255, 255, 255);
height: 50px;
font-size: 14pt;
}

QTableView::item{
border-style: none ;
height: px;
font-size: 14pt;
}

QTableView::item:select{
border: none;
background-color: rgba(255, 255, 255, 50);
}</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>135</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res-rc.qrc"/>
 </resources>
 <connections/>
</ui>
